### PR TITLE
feat: users with no edit and delete access

### PR DESCRIPTION
**Pull request by upalatucci**
Screenshot from 2023-01-24 15-27-21

image

### CODE REVIEW

File: locales/en/plugin__nmstate-console-plugin.json [hunk 1]
```diff
@@ -13,6 +13,8 @@
   "Archive Node network configuration policy": "Archive Node network configuration policy",
   "Archiving will remove the policy from all nodes, are you sure you want to archive this policy?": "Archiving will remove the policy from all nodes, are you sure you want to archive this policy?",
   "Available": "Available",
+  "Can not delete in view-only mode": "Can not delete in view-only mode",
```
Comment (pcbailey) [inline locales/en/plugin__nmstate-console-plugin.json]: 'Can not' is technically acceptable, but the recommended spelling is 'Cannot'.

Comment (upalatucci) [reply locales/en/plugin__nmstate-console-plugin.json]: Great thanks :-) Fixed

File: src/policies/actions/PolicyActions.tsx [hunk 2]
```diff
@@ -31,6 +34,16 @@ const PolicyActions: React.FC<PolicyActionsProps> = ({ policy, isKebabToggle })
   const [isEditModalOpen, setIsEditModalOpen] = React.useState(false);
   const [isDeleteModalOpen, setIsDeleteModalOpen] = React.useState(false);
 
+  const [canUpdatePolicy] = useAccessReview({
+    verb: 'update',
+    resource: NodeNetworkConfigurationPolicyModel.plural,
+  });
+
+  const [canDeletePolicy] = useAccessReview({
+    verb: 'update',
```
Comment (hstastna) [inline src/policies/actions/PolicyActions.tsx]: Shouldn't be here 'delete' verb ? Otherwise it looks totally the same as the lines above..

**Review by hstastna — COMMENTED** [review]
One more thing and then IMO we are good to go! ;)

File: src/policies/actions/PolicyActions.tsx (hunk 2 above)
Comment (upalatucci) [reply src/policies/actions/PolicyActions.tsx]: right  :D

**Review by hstastna — APPROVED** [review]

**Comment by hstastna** [comment]
@upalatucci  just a note for followup PR improvements:
`canUpdatePolicy ?? false` ;)

**Pull request merged**